src/components/MyEvents.js

```diff
@@ -412,20 +468,42 @@ const MyEvents = () => {
                             {selectedEvent.status === "Active" && <Button onClick={handleEditEvent}>Edit Event</Button>}
                         </DialogActions>
                     </Dialog>
+
+                    {/* Get event partipants dialog*/}
+                    < Dialog open={isParticipantsListOpen} onClose={handleCloseParticipantsDialog}>
+                        <DialogTitle>Participants</DialogTitle>
+                        <DialogContent>
+                            <div class="container">
+                                {selectedEventParticipants.map((event) => (
+                                    <ul>
+                                        {event.participantName} <br />
+                                    </ul>
+                                ))}
+                            </div>
+                        </DialogContent>
+                        <DialogActions>
+                            <Button onClick={handleCloseParticipantsDialog}>Close</Button>
+                        </DialogActions>
+                    </Dialog>
                 </div>
-            )}
-
-            {showCancelAlertMessage && (
-                <Alert severity="success">
-                    Event cancelled.
-                </Alert>
-            )}
-
-            {showEditAlertMessage && (
-                <Alert severity="success">
-                    Event successfully edited.
-                </Alert>
-            )}
+            )
+            }
+
+            {
+                showCancelAlertMessage && (
+                    <Alert severity="success">
+                        Event cancelled.
+                    </Alert>
+                )
+            }
+
+            {
+                showEditAlertMessage && (
+                    <Alert severity="success">
+                        Event successfully edited.
+                    </Alert>
+                )
+            }
```

Comment:
lines 489 to 506 have their brackets separated. Please replace it with the following:
```
            )};

            {showCancelAlertMessage && (
                <Alert severity="success">
                    Event cancelled.
                </Alert>
            )};

            {showEditAlertMessage && (
                <Alert severity="success">
                    Event successfully edited.
                </Alert>
            )};
```
sorry, very nitpicky 😖 if you have indentation issues I believe the keyboard shortcut on VsCode is `Alt` + `Shift` + `F`.